packages/cli-kit/src/private/node/components/Banner.tsx

```diff
@@ -0,0 +1,34 @@
+import {Box, Text, useStdout} from 'ink'
+import React from 'react'
+
+export type BannerType = 'success' | 'error' | 'warning' | 'info'
+
+interface Props {
+  type: BannerType
+  fullWidth?: boolean
+}
+
+function typeToColor(type: Props['type']) {
+  return {
+    success: 'green',
+    error: 'red',
+    warning: 'yellow',
+    info: 'white',
+  }[type]
+}
+
+const Banner: React.FC<Props> = ({type, fullWidth = false, children}) => {
+  const {stdout} = useStdout()
+  const width = fullWidth ? stdout!.columns : Math.min(60, stdout!.columns)
```

Comment:
Instead of using the full width for fatal errors, what about adjusting it to the longest line from the content? It looks a bit ugly in full screen. We could do the same for all the boxes to avoid too much empty space.

![06-36-y33w0-zmkk0](https://user-images.githubusercontent.com/14979109/194327370-01464b8e-b7ed-4bc8-b8ca-80dc0d4f830b.png)

Reply:
I've thought about this and couldn't find a good way to do it with `flex-box`. I'll need to dig deeper, but I agree the outcome you describe would be ideal. The only problem I can see is having a bunch of boxes on top of the other of slightly different widths. Not sure if that would look pleasing to the eye.

Reply:
Don't worry, I think it's ok for now, it was just a nitpicking suggestion.

You are right that having boxes with different sizes would be weird, but maybe we can define a minimum and just make it bigger when the content is too long. What do you think, @MeredithCastile?

Reply:
This has been reworked to be two thirds of the width of the terminal with a minimum width.